core/src/main/java/org/apache/accumulo/core/client/impl/TabletServerBatchWriter.java

```diff
@@ -733,7 +733,7 @@ else if (Tables.getTableState(context.getInstance(), table) == TableState.OFFLIN
     void queueMutations(final MutationSet mutationsToSend) throws InterruptedException {
       if (null == mutationsToSend)
         return;
-      binningThreadPool.execute(new Runnable() {
+      binningThreadPool.execute(Trace.wrap(new Runnable() {
```

Comment:
There's actually a TraceRunnable class which would do the same thing (perhaps a bit more concisely)